### ISSUE TITLE
Site Very Slow

### ISSUE THREAD

**Issue by scratchpads** [NaturalHistoryMuseum/scratchpads2#5748]
- Added by [Anna](http://icgf.myspecies.info/user/36)
- Scratchpad: [Κέντρο Απογραφής της Πανίδας της Ελλάδας - Greek Fauna Documentation Center](http://icgf.myspecies.info)

### Description:

Hello, 

Our site has been really slow over the holidays. Trying to open pages, edit them and edit terms.....

Is it a general problem with everybody's site?

**Comment by jpsullivan65** [comment]
Yes, it is a problem on mine.  Is it too much to expect a response from someone?

**Comment by jrdh** [comment]
Hi Anna and @jpsullivan65, we're looking into the performance issues right now and I'll update this issue as soon as I have more information. Apologies for the inconvenience.

**Comment by jrdh** [comment]
We believe we have now got to the bottom of the performance problems though we will confirm this tomorrow after more testing to be sure. Apologies for the inconvenience and thank you for your patience.
